src/main/java/com/uid2/optout/tool/OptOutLogTool.java

```diff
@@ -593,15 +593,18 @@ private void runGc() throws CloudStorageException {
         if (!deleteExpiredEnabled) return;
 
         String bucket = config.getString(Const.Config.OptOutS3BucketProp);
+        LOGGER.info("Listing Objects in bucket: " + bucket);
         ICloudStorage cloudStorage = wrapCloudStorageForOptOut(CloudUtils.createStorage(bucket, config));
         FileUtils utils = new FileUtils(config);
 
         List<String> expiredLogs = new ArrayList<>();
-        String cloudOptOutFolder = config.getString(Const.Config.OptOutS3FolderProp);
+        String cloudOptOutFolder = config.getString(Const.Config.OptOutS3FolderProp) + "delta/";
```

Comment:
maybe NIT: can we make it clear that "we want to clean delta only", instead of setting "cloudOptOutFolder = */delta"?